testsuite/tests/lib-uchar/test.ml

```diff
@@ -0,0 +1,108 @@
+(***********************************************************************)
+(*                                                                     *)
+(*                                OCaml                                *)
+(*                                                                     *)
+(*                           Daniel C. Buenzli                         *)
```

Comment:
I realize this is not centered but it's the first time we get a compiler hacking session with proper wine.